src/components/AuthPanel.tsx

```diff
@@ -95,6 +95,7 @@ const AuthPanel: React.FC<AuthPanelProps> = ({
         color: ${theme.error};
         font-size: 13px;
         text-align: center;
+        white-space: pre-line;
```

Comment:
Just curious about general code structure - do you know why this CSS is in a .tsx file rather than being referenced from a stylesheet file?

Reply:
I think it's because we use the styled-components library. The code here defines a new `ErrorMessage` component which inherits from a `span` but has these extra styles.

Reply:
Might need to look into this styled-components thing, don't think I've seen it before